google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -400,154 +402,6 @@ def _delete_cell(args, _):
       print('Failed to delete table %s: %s' % (args['name'], e))
 
 
-def _dryrun_cell(args, cell_body):
-  """Implements the BigQuery cell magic used to dry run BQ queries.
-
-   The supported syntax is:
-   %%bq dryrun [-q|--sql <query identifier>]
-   [<YAML or JSON cell_body or inline SQL>]
-
-  Args:
-    args: the argument following '%bq dryrun'.
-    cell_body: optional contents of the cell interpreted as YAML or JSON.
-  Returns:
-    The response wrapped in a DryRunStats object
-  """
-  query = _get_query_argument(args, cell_body, google.datalab.utils.commands.notebook_environment())
-
-  if args['verbose']:
-    print(query.sql)
-  result = query.execute_dry_run(dialect=args['dialect'], billing_tier=args['billing'])
-  return google.datalab.bigquery._query_stats.QueryStats(total_bytes=result['totalBytesProcessed'],
-                                              is_cached=result['cacheHit'])
-
-
-def _udf_cell(args, js):
```

Comment:
I think we should keep the current udf and pipeline code until the replacement is ready.

I.E. Add the new functionality and remove the old in one atomic change.

That will make it much easier for us to compare the new bits vs. the old to make sure we don't forget anything in the transition.